spec/integration/monorepo/before/Frameworks/d/D.podspec

```diff
@@ -37,7 +37,11 @@ Pod::Spec.new do |s|
   s.app_spec 'App' do |as|
     as.source_files = 'App/**/*.{h,m,swift}'
 
-    as.pod_target_xcconfig = { 'TARGETED_DEVICE_FAMILY' => '2' }
+    as.pod_target_xcconfig = {
```

Comment:
Worth a target in here that sets the triple also?

Reply:
Nice